### DISCUSSION TITLE
Running bookmarklets on webstorex

### DISCUSSION THREAD

**Discussion by BWUyguygd8ud** [3kh0/ext-remover#968] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Anyone know any methods to run bookmarklets on the chrome webstore/webstorex if bookmarklets are blocked

**Comment by AshtonDavies** [comment]
https://github.com/3kh0/ext-remover/discussions/845

**Reply by BWUyguygd8ud** [reply]
i did the steps but it didnt work

**Reply by BWUyguygd8ud** [reply]
![Uploading Screenshot 2024-03-08 9.42.31 PM.png…]()

**Reply by BWUyguygd8ud** [reply]
This site can’t be reachedchromewebstore.google.com refused to connect.
Try:

Checking the connection
[Checking the proxy and the firewall](chrome-error://chromewebdata/#buttons)
ERR_CONNECTION_REFUSED

**Reply by BWUyguygd8ud** [reply]
thats the error i get @AshtonDavies

**Reply by AshtonDavies** [reply]
Whatever you're doing to upload screenshots isn't working.

**Reply by BWUyguygd8ud** [reply]
im just taking screen shots and clicking that thing that says paste,drop, or click to add files

**Comment by BWUyguygd8ud** [comment]
![Screenshot 2024-03-08 9 42 31 PM](https://github.com/3kh0/ext-remover/assets/144839558/8febdb6a-001b-4b25-a19b-0aa06a9b8416)

**Comment by BWUyguygd8ud** [comment]
there @AshtonDavies

**Comment by AshtonDavies** [comment]
I think I've fixed it now.

**Reply by BWUyguygd8ud** [reply]
every time i open it i get
Invalid session

The session is invalid. You have to access KProxy main page to use it.

**Reply by BWUyguygd8ud** [reply]
What do i do @AshtonDavies

**Reply by BWUyguygd8ud** [reply]
@AshtonDavies i click the https://tiny.cc/bookmarkletexec and it takes me to a kproxy invalid session error and i click ok and try to open it again and its just the same thing but if i click ok then surf it takes me to like an unblocked google search bar and idk what to do to run bookmarklets

**Reply by AshtonDavies** [reply]
> @AshtonDavies i click the https://tiny.cc/bookmarkletexec and it takes me to a kproxy invalid session error and i click ok and try to open it again and its just the same thing but if i click ok then surf it takes me to like an unblocked google search bar and idk what to do to run bookmarklets

After you click ok, it should work when you repeat the process the second time. Otherwise, idk how to help.

**Reply by BWUyguygd8ud** [reply]
idk everytime i open it it just says invalid session

**Reply by BWUyguygd8ud** [reply, marked as answer]
OH I FIGURED IT OUT

**Reply by BWUyguygd8ud** [reply]
oh nvm

**Reply by S-PScripts** [reply]
OP inactive, marking answer until they return (ik they said "nvm")
